app/routes.tsx

```diff
@@ -134,21 +130,12 @@ export const routes = createRoutesFromElements(
       </Route>
 
       <Route path="system" element={<SystemLayout />} loader={SystemLayout.loader}>
-        <Route
-          element={<SilosPage />}
-          loader={SilosPage.loader}
-          handle={makeCrumb('Silos', pb.silos())}
-        >
+        <Route {...SilosPage} handle={makeCrumb('Silos', pb.silos())}>
```

Comment:
We could export handle from the module too, but I think that would make it impossible to reason about the crumbs by looking at the config. The components and loaders don’t have this problem.